src/main/java/com/avail/descriptor/ContinuationDescriptor.java

```diff
@@ -557,37 +582,67 @@ public static A_Continuation createContinuationExceptFrame (
 		return continuation;
 	}
 
+	/** The {@link CheckedMethod} for
```

Comment:
I'm going to leave it this ugly way for now, because the Java -> Kotlin converter _always_ messes up links that contain line breaks.  I'd rather have it ugly for a while than have to do extra work after conversion.